src/utils/__tests__/get-sorted-nodes-by-import-order.spec.ts

```diff
@@ -179,8 +182,13 @@ test('it returns all sorted nodes with builtin specifiers at the top', () => {
 test('it returns all sorted nodes with custom third party modules and builtins at top', () => {
     const result = getImportNodes(code);
     const sorted = getSortedNodesByImportOrder(result, {
-        importOrder: ['^a$', '<THIRD_PARTY_MODULES>', '^t$', '^k$', '^[./]'],
-        importOrderCombineTypeAndValueImports: true,
```

Comment:
I think this is lost if we don't specify a `importOrderTypeScriptVersion`, right?  I don't think you meant to change that behavior in this test.